pkg/storaged/stratis-details.jsx

```diff
@@ -185,10 +197,29 @@ export const StratisPoolDetails = ({ client, pool }) => {
                       pool.ClevisInfo[0] && // pool has consistent clevis config
                       (!pool.ClevisInfo[1][0] || pool.ClevisInfo[1][1][0] == "tang")); // not bound or bound to "tang"
     const tang_url = can_tang && pool.ClevisInfo[1][0] ? JSON.parse(pool.ClevisInfo[1][1][1]).url : null;
+    const blockdevs = client.stratis_pool_blockdevs[pool.path] || [];
 
     const forced_options = ["x-systemd.requires=stratis-fstab-setup@" + pool.Uuid + ".service"];
     const managed_fsys_sizes = client.features.stratis_managed_fsys_sizes && !pool.Overprovisioning;
 
+    function grow_blockdevs() {
+        return for_each_async(blockdevs, bd => pool.GrowPhysicalDevice(bd.Uuid));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230825-082034-fda8c7cb-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)